app/services/pro_membership_manager.rb

```diff
@@ -44,8 +44,23 @@ def cancel!
   # Renew the user's pro membership. Make sure the plan they are on is recurring
   # first. If we don't succeed in charging the user then send a dunning email
   # and let their membership expire.
-  def renew!
-    # TODO we don't actually need this until one month after we launch PRO.
+  def renew!(attempt_number: 0)
```

Comment:
My assumption is that the number will be crammed into the template, not that it'll use a different template for each.

The attempts are already capped by the worker tbh